File: SampleWebAssembly/Pages/CallNetFromJS.razor.cs
using System.Runtime.InteropServices.JavaScript;
using System.Runtime.Versioning;

namespace SampleBlazorWebAssembly.Pages
{
    [SupportedOSPlatform("browser")]
    public partial class CallNetFromJS
    {
        [JSImport("setMessage", "CallNetFromJS")]
        internal static partial void SetWelcomeMessage();

        [JSExport]
        internal static string GetMessageFromDotnet()
        {
            return "Olá do Blazor!";
        }
    }
}

File: SampleWebAssembly/Pages/CallNetFromJS.razor.js
export async function setMessage() {
    const { getAssemblyExports } = await globalThis.getDotnetRuntime(0);
    var exports = await getAssemblyExports("SampleBlazorWebAssembly.dll");
    console.log(exports);
    document.getElementById("result").innerText =
        exports.SampleBlazorWebAssembly.Pages.CallNetFromJS.GetMessageFromDotnet();
}